build_docs.pl

```diff
@@ -531,7 +555,7 @@ sub init_target_repo {
     my ( $repos_dir, $temp_dir, $reference_dir ) = @_;
 
     my $target_repo = ES::TargetRepo->new(
-        dir         => $repos_dir,
+        git_dir     => $repos_dir->subdir('target_repo.git'),
```

Comment:
I this case we really prefer to keep this in a stable place because it prevents us from having to pull as much.

Long before I got to it `build_docs --all` sacrificed being run in parallel for less git traffic. For the most part we don't want to run it in parallel anyway because its job is to commit to a particular repository.

Reply:
I see. Thanks for the explanation!  Then it doesn't really matter 😄